config/opensearch_dashboards.yml

```diff
@@ -223,3 +223,12 @@
 # Set the value of this setting to true to start exploring wizard
 # functionality in Visualization.
 # wizard.enabled: false
+
+# Set the value of this setting to true to enables the experimental multiple data source
+# support feature. Use with caution.
+#data_source.enabled: false
+# Set the value of these settings to custermize crypto materials to encryption saved credentials
+# in data sources.
```

Comment:
Can/should we link to documentation for these settings? It's not at all clear from this file alone what valid values here might look like.

Reply:
I don't see other settings has reference to document in the template. there is another PR which provides document of all settings in README https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2448